S23/akumar2709/main.py

```diff
@@ -0,0 +1,32 @@
+import csv
+
+#function converts word to lower case and provides special rules for irish and turkish since they are not covered by the book
+def toLower(input_string : str, language: str) -> str:
+  irish_vowels = "AEIOUÁÉÍÓÚ"
+  tilda_ord = 771
```

Comment:
There is something more general going on here that you'll want to handle in general, not just for this one combining diacritics.  We'll discuss in class when we get into Unicode more deeply, or you can stop by office hours if you're curious.